models/ht_neuron.cpp

```diff
@@ -49,8 +49,7 @@ RecordablesMap< ht_neuron >::create()
     Name( "Theta" ), &ht_neuron::get_y_elem_< ht_neuron::State_::THETA > );
   insert_(
     Name( "g_AMPA" ), &ht_neuron::get_y_elem_< ht_neuron::State_::G_AMPA > );
-  insert_(
-    Name( "g_NMDA" ), &ht_neuron::get_y_elem_< ht_neuron::State_::G_NMDA > );
+  insert_( Name( "g_NMDA" ), &ht_neuron::get_g_NMDA_ );
```

Comment:
Yes, I agree that ``g_NMDA`` should be the variable recorded, and ``G_NMDA_TIMECOURSE`` looks good!